lib/common-types.ts

```diff
@@ -1124,6 +1125,10 @@ export type MRActions = {
       params: GetReleaseParams & { actionId: string }
       return: ReleaseAction
     }
+    getMany: {
+      params: GetSpaceEnvironmentParams & { query: ReleaseActionQueryOptions }
+      return: Collection<ReleaseAction, ReleaseActionProps>
```

Comment:
I find it strange that here we use type `Collection` while everywhere else we use `CollectionProp`. Should we align? (relates to other functions of releases as well)

Reply:
Here's [an open PR](https://github.com/contentful/contentful-management.js/pull/1299) that will be the next breaking change in case you maybe want to change it for in the major version

Reply:
Interestingly, `CollectionProp` is the "pure" collection object, `Collection` is what it returns here since we wrap it with `wrapCollection` which runs the `toPlainObject()` function. The others might actually be incorrect :|

Reply:
I might actually just use `CollectionProp` since every other endpoint uses it like such 


(source of Collection: https://github.com/contentful/contentful-management.js/blob/79c4e518b8fb3acd22d783db7f98a3284e05271f/lib/common-utils.ts#L15-L16 - here we add a new prop `toPlainObject` that `CollectionProp` doesn't have)

Reply:
I don't see any line of code in this PR calling wrapCollection() method

Reply:
oh I see. That is for env api, not for plain client

Reply:
`plainClient` uses a different file though: https://github.com/contentful/contentful-management.js/blob/740dc15534a672369af4116d9407b4210d26b73e/lib/plain/common-types.ts#L558 (which uses `CollectionProp` correctly)

Reply:
worth discussing with the rest of the teams about it, but looks like you might be right